src/tablo/Tablo.js

```diff
@@ -62,24 +64,25 @@ const { maybe } = t;
   onScrollStart: maybe(t.Function),
   onScrollEnd: maybe(t.Function),
   children: t.ReactChildren,
+  rowClassNameGetter: maybe(t.Function),
 
   // private
   scrollToRow: maybe(t.Integer),
   onRowClick: maybe(t.Function),
-  rowClassNameGetter: maybe(t.Function),
   onColumnResizeEndCallback: maybe(t.Function),
   isColumnResizing: maybe(t.Boolean)
 })
 export default class Tablo extends React.Component {
 
   static defaultProps = {
+    rowClassNameGetter: constant(''),
```

Comment:
please don't.... it's like the ugliest name ever for this utility....and `() => ''` is even shorter.....